Makefile

```diff
@@ -56,7 +56,7 @@ check-venv:
 	fi
 
 install-user: venv-create
-	. $(VENV_ACTIVATE_FILE); $(PIP_WRAPPER) install --upgrade pip setuptools wheel
+	. $(VENV_ACTIVATE_FILE); $(PIP_WRAPPER) install --upgrade pip setuptools==58.0.1 wheel==0.37.0
```

Comment:
Can you please pin `pip` as well? In my experience that's the most actively developed one, and they regularly introduce breaking changes